v1/nix/modules/drv-parts/pip/default.nix

```diff
@@ -3,9 +3,8 @@
   lib,
   dream2nix,
   ...
-}: let
+} @ topLevel: let
```

Comment:
As discussed, I don't think those changes are a good idea in the long run. I think a solution similar to the usage of `setuptools_scm` in `apache-airflow` in https://github.com/nix-community/dream2nix/pull/529 could help with your setuptools issue in a cleaner way?

Reply:
Reverted the changes